chaoslib/litmus/platform/gke/disk_loss.yml

```diff
@@ -1,31 +1,44 @@
-- name: Wait for the specified ramp time before injecting chaos
-  wait_for: timeout="{{ ramp_time }}"
-  when: "ramp_time is defined and ramp_time != ''"
+- block:
+
+    - debug:
+        msg: "***** Waiting for the ramp interval of {{ ramp_time }}s *****"
+
+    - name: "[Ramp]: Waiting for the specified ramp time before injecting chaos"
+      wait_for: timeout="{{ ramp_time }}"
+
+  when: "ramp_time is defined and ramp_time != ''" 
 
 ## RECORD EVENT FOR CHAOS INJECTION
-- include_tasks: /utils/common/generate-kubernetes-chaos-events.yml
+- name: "[Event]: Generating an Event for ChaosInjection"
+  include_tasks: /utils/common/generate-kubernetes-chaos-events.yml
   vars:
     stage: "ChaosInject"
     exp_pod_name: "{{ chaos_pod_name }}"
     engine_ns: "{{ c_ns }}"
     message: "Injecting {{ c_experiment }} chaos on {{ disk_name }} disk"
   when: "c_engine is defined and c_engine != ''"
 
-- name: Detaching the disk
+- name: "[Inject]: Detaching the {{ disk_name }} disk from {{ node_name }} node"
   shell: gcloud compute instances detach-disk {{ node_name }} --disk {{ disk_name }} --zone {{ zone_name }}
 
-- name: chaos injection for {{ c_duration }}s
+- name: "[Wait]: Waiting for {{ c_duration }}s after chaos injection"
```

Comment:
`"[Wait]: Waiting for chaos duration of {{ c_duration }}s after chaos injection"`